h5p-bildetema/src/components/Language/Language.module.scss

```
@@ -0,0 +1,76 @@
+.language {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  gap: 1.5rem;
+}
+
+.input_container {
+  border: 1px solid red;
+  position: relative;
+  padding-left: 1rem;
+  cursor: pointer;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
+}
+
+.input_container input {
+  position: absolute;
+  opacity: 0;
+  cursor: pointer;
+  height: 0;
+  width: 0;
+}
+
+.checkmark {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
```

This can in most cases be shortened to `inset: 0`.

```suggestion
  inset: 0;
```